Documentation/Books/Manual/Programs/Arangorestore/ParallelRestore.md

```diff
@@ -1,2 +1,198 @@
-Parallel Restore Procudure
+Parallel Restore Procedure
 ==========================
+
+The following parallel restore procedure is recommended to speed up the
+performance of [_arangorestore_](../Arangorestore/README.md) in a
+cluster environment.
+It is assumed that a cluster environment is running and a `dump` with
+[_arangodump_](../Arangodump/README.md) has been already been created.
+
+### Allocate dump Directory
+
+The first step is to copy the `dump` directory to all machines where
+coordinators are running. 
+
+{% hint 'info' %}
+This step is not strictly required as the backup can be restored over the
+network. However, if the restore is executed locally the restore speed is
+significantly improved.
+{% endhint %}
+	
+### Restore Collection Structure
+
+The collection structure has to be restored from exactly one coordinator (any
+coordinator can be used) with the following command:
+
+```
+arangorestore
+  --server.endpoint <endpoint-of-a-coordinator>
+  --server.database <database-name> 
+  --server.password <password> 
+  --import-data false 
+  --input-directory <dump-directory>
+```
+
+
+The option `--import-data false`  tells _arangorestore_ to restore only the
+collection structure and no data.
+
+### Set Replication Factor to 1
+
+It is necessary to set the the replication factor to 1 before importing any
+data. Run the following command from exactly one coordinator (any coordinator
+can be used):
+
+```
+echo 'db._collections().filter(function(c) { return c.name()[0] !== "_"; })
+.forEach(function(c) { print("collection:", c.name(), "replicationFactor:",
+c.properties().replicationFactor); c.properties({ replicationFactor: 1 }); });'
+| arangosh
```

Comment:
If this is supposed to be copy-paste-able, then it needs to be written on a single line (there is no universal way to escape line breaks for formatting purposes across shells on different OSs).

Note that this will not work on Windows: echo outputs everything literally, which includes the single quote marks.

Reply:
Yes it should be copy-paste-able. I tried to make it a bit more readable but I guess we have it has to be one line in this case.

Reply:
We should probably find a better solution mid-term, so that we can format it to span multiple lines but with a button to copy to clipboard as single line.